src/Contrib/Authentication/OpenIdConnect/Masa.Contrib.Authentication.OpenIdConnect.EFCore/Repositories/ApiResourceRepository.cs

```diff
@@ -41,6 +41,7 @@ public async Task<PaginatedList<ApiResource>> GetPaginatedListAsync(int page, in
                         .Include(apiResource => apiResource.Properties)
                         .Include(apiResource => apiResource.ApiScopes)
                         .ThenInclude(apiScope => apiScope.ApiScope)
+                        .AsSplitQuery()
```

Comment:
I don't think it needs split query, you just need control left join and inner join.